libs/ui/select/brain/src/lib/brn-select-content.component.ts

```diff
@@ -215,18 +211,18 @@ export class BrnSelectContentComponent implements AfterViewInit {
 	}
 
 	public moveFocusUp() {
-		this.viewport.nativeElement.scrollBy({ top: -100, behavior: 'smooth' });
-		if (this.viewport.nativeElement.scrollTop === 0) {
-			this.scrollUpBtn.stopEmittingEvents();
+		this.viewport().nativeElement.scrollBy({ top: -100, behavior: 'smooth' });
```

Comment:
We have these magic numbers -100, 100, 50. Thoughts on making those constants with a descriptive name? Not a blocker though